operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-svc.yaml

```diff
@@ -25,6 +25,9 @@ spec:
   {{- if semverCompare ">= 1.22-0" (include "mimir.kubeVersion" .) }}
   internalTrafficPolicy: {{ .Values.store_gateway.service.internalTrafficPolicy }}
   {{- end }}
+  {{- if .Values.store_gateway.service.trafficDistribution }}
+  trafficDistribution: {{ .Values.store_gateway.service.trafficDistribution }}
```

Comment:
similar to the ingester, this Service   object isn't responsible for any of the traffic on the read path. i would clarify this in the values.yaml documentaiton